hazelcast/src/main/java/com/hazelcast/instance/impl/MobyNames.java

```diff
@@ -352,7 +352,7 @@ public final class MobyNames {
             // Pierre de Fermat pioneered several aspects of modern mathematics. https://en.wikipedia.org/wiki/Pierre_de_Fermat
             "fermat",
 
-            // Enrico Fermi invented the first nuclear reactor. https://en.wikipedia.org/wiki/Enrico_Fermi.
+            // Enrico Fermi invented the first nuclear hazelcast.tpc. https://en.wikipedia.org/wiki/Enrico_Fermi.
```

Comment:
:bomb:

Reply:
I will address this after merge under https://github.com/hazelcast/hazelcast/issues/23775.